ClientLib/src/main/java/com/o3dr/services/android/lib/drone/companion/solo/tlv/SoloPanoOptions.java

```diff
@@ -0,0 +1,157 @@
+package com.o3dr.services.android.lib.drone.companion.solo.tlv;
+
+import android.os.Parcel;
+import android.support.annotation.IntDef;
+
+import java.nio.ByteBuffer;
+
+/**
+ * Created by phu on 6/24/16.
+ */
+public class SoloPanoOptions extends SoloShotOptions {
```

Comment:
Because the ShotState inside the Solo app only accepts SoloShotOptions-type messages. 

SoloShotOptions could be more barebones than what it is now to be a true subset of SoloPanoOptions. But this required changing more code than needed.